chcemvediet/apps/inforequests/cron.py

```diff
@@ -217,11 +219,14 @@ def publish_inforequests():
         try:
             for branch in inforequest.branches:
                 action = branch.last_action
-                if (action.deadline and
-                        action.deadline.snooze_calendar_days_behind < DAYS_TO_PUBLISH_INFOREQUEST):
+                days_to_publish = DAYS_TO_CLOSE_INFOREQUEST + DAYS_TO_PUBLISH_INFOREQUEST
+                if action.deadline is None:
+                    if action.created > utc_now() - timedelta(days=DAYS_TO_PUBLISH_INFOREQUEST):
+                        break
+                elif action.deadline.snooze_calendar_days_behind < days_to_publish:
                     break
             else:
-                # Every inforequest that has been closed for at least 60 WD.
+                # Every inforequest that has been closed for at least DAYS_TO_PUBLISH_INFOREQUEST WD
```

Comment:
@martinmacko47 Ak akcia inforequestu ma deadline, tak sa porovnavaju WD - pracovne dni, ak nema deadline, tak sa porovnavaju CD - kalendarne dni. Je to zle, treba tento stav tiez osetrit?

Reply:
@viliambalaz Sorry, komentar pri zatvarani inforequestov bol napisany zle. V podmienke sa porovnava `action.deadline.snooze_calendar_days_behind`. Ked si pozries implementaciu `snooze_calendar_days_behind`, tak ten pocet sa rata ako `(self.snooze_date - local_today()).days`, cize pocet kalendarnych dni, nie pracovnych dni. Pocet pracovnych dni by sa porovanal ak by sa porovnalo `snooze_workdays_behind` nie `snooze_calendar_days_behind`.

Cize pls iba preformuluj komentar aj v `close_inforequests` aj v `publish_inforequests` ze ide o kalendarne dni, nie pracovne.